src/Table/TableCell.js

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react'
+import propTypes from 'prop-types'
+import css from 'styled-jsx/css'
+
+import { TableCellText } from './TableCellText.js'
+
+const tableCellStyles = css`
+    td {
+        border-bottom: 1px solid #e8edf2;
+        padding: 0 12px;
+        font-size: 14px;
+    }
+
+    div {
+        min-height: 45px;
+    }
+
+    :global(tfooter) div {
+        min-height: 36px;
+    }
+`
+
+export const TableCell = ({ children, colSpan, rowSpan }) => (
+    <td colSpan={colSpan} rowSpan={rowSpan}>
+        <div>
+            <TableCellText>{children}</TableCellText>
```

Comment:
Why are we restricting ourselves to phrasing content here?

A `td` can contain flow content (e.g. `div`, `h1`, `p`) so the `children` should be valid for any [flow content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Flow_content).

As we force all the `children` into a `TableCellText`, which in turn puts the `children` inside of a `span`, we are restricting the `children` of `TableCell` to only be [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content). This is unnecessarily restrictive.

See permitted content table:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td

I think we should drop the `TableCellText` entirely.